### PR TITLE
Inline edit - UI fixes for side-by-side rendering

### PR DESCRIPTION
**Pull request by Krzysztof-Cieslak**
CC: @hediet

### CODE REVIEW

File: src/vs/editor/contrib/inlineEdit/browser/inlineEditSideBySideWidget.ts [hunk 1]
```diff
@@ -281,13 +296,13 @@ class InlineEditSideBySideContentWidget extends Disposable implements IOverlayWi
 		this._register(this._previewEditorObs.setDecorations(this._modifiedDecorations));
 
 		this._register(autorun(reader => {
+			this._previewEditorObs.model.read(reader);
```
Comment (hediet) [inline src/vs/editor/contrib/inlineEdit/browser/inlineEditSideBySideWidget.ts:299]: I'm not sure you need this dependency

File: src/vs/editor/contrib/inlineEdit/browser/inlineEditSideBySideWidget.ts [hunk 2]
```diff
@@ -281,13 +296,13 @@ class InlineEditSideBySideContentWidget extends Disposable implements IOverlayWi
...
 			const width = this._previewEditorObs.contentWidth.read(reader);
 			const lines = this._text.get().split('\n').length - 1;
```
Comment (hediet) [inline src/vs/editor/contrib/inlineEdit/browser/inlineEditSideBySideWidget.ts:301]: Instead, `_text` should be `.read(reader)`.

File: src/vs/editor/contrib/inlineEdit/browser/ghostTextWidget.ts [hunk 3]
```diff
@@ -158,22 +158,13 @@ export class GhostTextWidget extends Disposable {
 			});
 		}
 
+		console.log('isPureRemove', uiState.isPureRemove, uiState.range, uiState.isSingleLine);
```
Comment (hediet) [inline src/vs/editor/contrib/inlineEdit/browser/ghostTextWidget.ts:161]: This log is intentionally?

Comment (Krzysztof-Cieslak) [reply src/vs/editor/contrib/inlineEdit/browser/ghostTextWidget.ts:161]: Nope, good catch.

File: src/vs/editor/contrib/inlineEdit/browser/inlineEditSideBySideWidget.ts (hunk 1 above)
Comment (Krzysztof-Cieslak) [reply src/vs/editor/contrib/inlineEdit/browser/inlineEditSideBySideWidget.ts]: Yeah, I don't it was leftover after some experimentation. Deleted.

**Review by hediet — APPROVED** [review]

**Review by alexr00 — APPROVED** [review]

**Pull request merged**